src/Components/App/App.css

```diff
@@ -1,38 +1,49 @@
-.App {
-  text-align: center;
+.app {
+  display: flex;
+  flex-direction: column;
+  width: 40%;
+  align-items: center;
+  margin-inline: 30%;
+  margin-top: 15%;
+  border: 2px dotted rgb(30, 30, 174);
+  background-color: beige;
+  border-radius: 0 0 70px 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.addBtn {
+  padding: 5px 25px;
+  border-radius: 10px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.nextTD {
+  margin-top: 10px;
+  width: 300px;
+  margin-bottom: 10px;
+  padding: 10px;
+  border-radius: 12px;
+  font-size: 1.5rem;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.currentTD {
+  padding: 5px;
+  width: 300px;
+  border: none;
+  font-size: 0.9rem;
+  background-color: beige;
+  margin: 10px 0;
+}
+
+ul {
+  list-style: none;
 }
 
-.App-link {
-  color: #61dafb;
+li {
+  display: flex;
```

Comment:
![review-1](https://user-images.githubusercontent.com/89055502/190854795-5b9677de-c554-4e16-8bc3-8f1d8467839b.png)

- [ ] Kindly adjust the alignment of each todo on the horizontal axis. 
 
above is an image of how your todo app currently looks like in the browser, you can fix this by adding `align-items: center` to the `li` element selector in your `app.css` file

Reply:
I did check the alignment on this file, it looks like this on my system, still suggestion was applied, thank you
![Screen Shot 2022-09-17 at 08 50 19](https://user-images.githubusercontent.com/99816838/190862987-2eede7b2-1d99-4d71-8521-6a1f2fde1f9d.png)